core/src/components/cat-button/cat-button.scss

```diff
@@ -11,8 +11,10 @@
 .cat-button {
   font: inherit;
   display: flex;
+  flex: 1 1 auto;
```

Comment:
please revert this change. we will fix it separately

Reply:
done